src/test/java/htsjdk/samtools/filter/TagFilterTest.java

```diff
@@ -69,10 +69,10 @@ public void testTagFilter(final String testName, final String tag, final List<Ob
     private Object[][] getTagFilterTestData()
     {
         return new Object[][]{
-            {"Basic positive test", ReservedTagConstants.XN, Arrays.asList(1), 1, true},
-            {"Multi-value positive test", ReservedTagConstants.XN, Arrays.asList(1,2,3), 1, true},
-            {"Incorrect value negative test", ReservedTagConstants.XN, Arrays.asList(1), 2, false},
-            {"Null value negative test", ReservedTagConstants.XN, Arrays.asList(1), null, false} 
+                {"Basic positive test", ReservedTagConstants.XN, Arrays.asList(1), 1, true, false},
+                {"Multi-value positive test", ReservedTagConstants.XN, Arrays.asList(1,2,3), 1, false, true},
+                {"Incorrect value negative test", ReservedTagConstants.XN, Arrays.asList(1), 2, false, false},
+                {"Null value negative test", ReservedTagConstants.XN, Arrays.asList(1), null, true, true}

```

Comment:
please add positive tests with includeReads==true

Reply:
isnt the 2nd test a positive test with includeReads==true?

Reply:
could be, but there were 4 tests before your changes...so it woudl be good if you left those unchanges, and added some to test the new functionality...I was reacting to the fact that the total number of tests didn't increase.

Reply:
I mean i will add more tests but I dont think it is always the the case that new functionality requires "more" tests if you can change the current tests to test both new and old functionality